README.md

```diff
@@ -3,93 +3,154 @@
 [![Build Status](https://drone.fpfis.eu/api/badges/openeuropa/oe_webtools/status.svg?branch=master)](https://drone.fpfis.eu/openeuropa/oe_webtools)
 [![Packagist](https://img.shields.io/packagist/v/openeuropa/oe_webtools.svg)](https://packagist.org/packages/openeuropa/oe_webtools)
 
-Webtools integration for OpenEuropa.
+[Webtools](http://ec.europa.eu/ipg/services/interactive_services/index_en.htm) integration for OpenEuropa.
+Webtools are interactive services available to integrate in a website.
+
+**Table of contents:**
+
+- [Installation](#installation)
+- [Development setup](#development-setup)
+- [Contributing](#contributing)
+- [Versioning](#versioning)
+
+## Installation
+
+The recommended way of installing the OpenEuropa Webtools module is via [Composer][1].
+
+```bash
+composer require openeuropa/oe_webtools
+```
+
+### Enable the module
+
+In order to enable the module in your project run:
+
+```bash
+./vendor/bin/drush en oe_webtools
+```
+
+### OpenEuropa Webtools Analytics
+
+The Webtools module contains a submodule that provides a service for providing
+analytics information. For more information on how to use and configure this module,
+check out the module [README](modules/oe_webtools_analytics/README.md).
+
+### OpenEuropa Webtools Laco Service
+
+The Webtools module contains a submodule that provides a service for retrieving
+information about language coverage of entity resources. For more information on 
+how to use and test this module, check out the module [README](modules/oe_webtools_laco_service/README.md).
+
+### OpenEuropa Webtools Laco Widget
+
+The Webtools module contains a submodule that provides a widget which integrates
+with the Laco service. For more information on how to use and configure this module, 
+check out the module [README](modules/oe_webtools_laco_widget/README.md).
 
 ## Development setup
 
 You can build the test site by running the following steps.
 
 * Install all the composer dependencies:
 
-```
-$ composer install
+```bash
+composer install
 ```
 
 * Customize build settings by copying `runner.yml.dist` to `runner.yml` and
 changing relevant values, like your database credentials.
 
 * Setup test site by running:
 
-```
-$ ./vendor/bin/run drupal:site-setup
+```bash
```

Comment:
This should be removed like in other repositories we are now updating.

Reply:
Fixed.